clap_complete/examples/exhaustive.rs

```diff
@@ -186,6 +186,7 @@ fn cli() -> clap::Command {
                     .long("email")
                     .value_hint(clap::ValueHint::EmailAddress),
             ]),
+            clap::Command::new("escape-help").about("\\tab\t\"'"),
```

Comment:
Can you split this change out into its own commit so all of the completion script updates that happen from it don't clog up other commits?